api/views.py

```diff
@@ -234,6 +234,38 @@ def get_statistics(request):
 
     return Response(data=list(result))
 
+@api_view(["POST"])
```

Comment:
why POST when the method is called `get_user_statistics` ?

Reply:
We need to use POST to be able to pass a body with request details.

Reply:
makes sense